pkgs/by-name/au/audit/package.nix

```diff
@@ -36,6 +36,11 @@ stdenv.mkDerivation (finalAttrs: {
     hash = "sha256-NX0TWA+LtcZgbM9aQfokWv2rGNAAb3ksGqAH8URAkYM=";
   };
 
+  patches = [
```

Comment:
We shouldn't introduce a patch just so a test can run on a specific kernel. 

Instead, I propose we add the test to passthru [like I did for openssh](https://github.com/NixOS/nixpkgs/pull/277579) not building them by default when you build the package. I don't have a strong opinion whether we include this in a release.nix or whether it's enough to have it in `passthru` so that contributors can test it easily and ofborg tests it automatically.

Reply:
Implemented here: https://github.com/NixOS/nixpkgs/pull/453516

Considering the tests were just recently introduced, I don't think its worth it to add them to a jobset.